StigData/Processed/WindowsDefender-All-1.5.org.default.xml

```diff
@@ -0,0 +1,25 @@
+<!--
+    The organizational settings file is used to define the local organizations
+    preferred setting within an allowed range of the STIG.
+
+    Each setting in this file is linked by STIG ID and the valid range is in an
+    associated comment.
+-->
+<OrganizationalSettings fullversion="1.5">
+  <!-- Ensure ''V-75167'' -match '2|1'-->
+  <OrganizationalSetting id="V-75167" value="2" />
+  <!-- Ensure 'V-75237' -ge '0' -and 'V-75237' -le '7'-->
+   <OrganizationalSetting id="V-75237" value="0" />
```

Comment:
There's an extra space in front of this setting.

Reply:
Good catch.
Done.